client/a8c-for-agencies/sections/partner-directory/components/industries-selector.tsx

```diff
@@ -31,12 +31,18 @@ const IndustriesSelector = ( { setIndustries, industries }: Props ) => {
 	);
 
 	// Get the selected industries by label
-	const selectedIndustriesByLabel = industries.flatMap( ( slug ) => {
-		const key = slug as string;
-		const value = availableIndustries[ key ];
-		return value ? [ value ] : [];
-	} );
-
+	const selectedIndustriesByLabel = Array.from(
+		new Set(
+			industries.flatMap( ( slug ) => {
+				const key = slug as string;
+				const value =
+					key in oldIndustries
+						? availableIndustries[ oldIndustries[ key ] ]
+						: availableIndustries[ key ];
+				return value ? [ value ] : [];
```

Comment:
Can this be simplified to this line?

```
return availableIndustries[oldIndustries[key] || key];
```

Reply:
I have update it now: https://github.com/Automattic/wp-calypso/pull/95053/commits/2de9f73edbf90443a0e7c099baf86969b5c6b480